packages/web-frontend/src/components/View/SingleValueWrapper.js

```diff
@@ -26,14 +26,16 @@ import { formatDataValue } from '../../utils';
 
 export class SingleValueWrapper extends PureComponent {
   render() {
-    const { name, valueType, value, total, value_metadata } = this.props.viewContent;
-    const metadata = value_metadata || this.props.viewContent[`${name}_metadata`];
+    const { name, valueType, data, total, value_metadata: valueMetadata } = this.props.viewContent;
```

Comment:
Single value reports no longer returns just `value`. I changed this I can remove `viewJson` from the `report` endpoint, and also it can be consistent with how other dashboard item types receiving data